hcs/swagger.json

```diff
@@ -726,20 +893,6 @@
       },
       "title": "See ConsulAMAService.ListCluster"
     },
-    "hashicorp.cloud.consulama.ama.ResourceNotificationError": {
```

Comment:
This is scary.

Do we need to confirm with Paul/Tiago that this change of type `ResourceNotificationRequest.Error` is expected? 

I'm lead to believe this is just a type adjustment/rename, so wont consider it a blocker, but yeah, it's scary.

Reply:
Looks like there has been some modifications to this recently to support billing so this is intentional.